charts/caraml/values.yaml

```diff
@@ -41,6 +41,18 @@ global:
   mlflow:
     serviceName: merlin-mlflow
     externalPort: "80"
+  turing:
+    postgresqlDatabase: turing
+  xp:
```

Comment:
need to update the global-values file? Or should we even remove it?

Reply:
What do you mean?

Reply:
this file: https://github.com/caraml-dev/helm-charts/blob/main/charts/caraml/values-global.yaml it needs updates too?

Reply:
ah i see. Originally this file was meant to be used with the `-f values-global.yaml` flag when passing helm-install, but i suppose we don't need it anymore?

Reply:
yeah, we don't.